src/cell.h

```diff
@@ -257,12 +257,15 @@ struct Cell {
             str.Append(L"</cell>\n");
         } else if (format == A_EXPHTMLT) {
             wxString style;
-            if (text.stylebits & STYLE_BOLD) style += L"font-weight: bold;";
-            if (text.stylebits & STYLE_ITALIC) style += L"font-style: italic;";
-            if (text.stylebits & STYLE_FIXED) style += L"font-family: monospace;";
-            if (cellcolor != doc->Background())
+            if (parent ? (text.stylebits & STYLE_BOLD) != (parent->text.stylebits & STYLE_BOLD) : true)

```

Comment:
can be simplified to `!parent || (text.stylebits & STYLE_BOLD) != (parent->text.stylebits & STYLE_BOLD)`

Reply:
Ah thanks! (y)